meson.build

```diff
@@ -1,5 +1,5 @@
 project('harfbuzz', 'c', 'cpp',
-  meson_version: '>= 0.55.0',
+  meson_version: '>= 0.60.0',
```

Comment:
We intentionally don't depend on a too new version of meson.

Reply:
Then feel free to close.

Reply:
If @behdad is OK with raising the minimum version we can do it, we can also keep the minimum version of older meson versions will ignore the new option.

Reply:
Does it? From my limited testing, the build will fail if an unknown keyword is used (e.g. after renaming to `install_tags`).

Reply:
I prefer not. It's just over a year old.